src/UI/TransactionsTable/TransactionMethod.tsx

```diff
@@ -1,9 +1,9 @@
-import { UITransactionType } from 'components/TransactionsInterpreter/helpers/types';
 import React from 'react';
-import getTransactionMethod from 'components/TransactionsInterpreter/helpers/getTransactionMethod';
-import styles from './transactions-table.scss';
-import globalStyles from 'assets/sass/main.scss';
 import classNames from 'classnames';
+import globalStyles from 'assets/sass/main.scss';
```

Comment:
align imports here. move global styles below

Reply:
Can't. Eslint autoformatter is rearranging on save.